docs/examples/index.md

```diff
@@ -1,8 +1,113 @@
 ---
-title: Basic usage
-sidebar_label: Basic usage
+sidebar_label: Examples
+title: Examples
 ---
 
-This section contains examples of how to use Apify Client for Javascript.
+# Code examples
 
-There is nothing here yet.
+## Different ways how to pass an input
+
+The fastest way to get results from an Actor is to pass input directly to the `call` function.
+
+```javascript
+import { ApifyClient } from 'apify-client';
+
+// Client initialization with the API token
+const client = new ApifyClient({ token: 'MY_APIFY_TOKEN' });
+
+const actorClient = client.actor('apify/instagram-hashtag-scraper');
+
+// Run the actor and wait for it to finish up to 60 seconds. Input is not persisted for next runs.
+const runData = await actorClient.call({ hashtags: ['twitter'], resultsLimit: 20 }, { waitSecs: 60 });
+```
+
+To run multiple inputs with the same Actor, most convenient way is to create multiple [tasks](https://docs.apify.com/platform/actors/running/tasks) with different inputs.
+Task input is persisted on Apify platform when task is created.
+
+```javascript
+import { ApifyClient } from 'apify-client';
+
+// Client initialization with the API token
+const client = new ApifyClient({ token: 'MY_APIFY_TOKEN' });
+
+const socialsHashtags = ['facebook', 'twitter', 'instagram'];
+
+// Multiple input schemas for one Actor can be persisted in tasks. Tasks are saved in the Apify platform and can be run multiple times.
+const socialsTasks = socialsHashtags.map((hashtag) => client.tasks().create({
+    actId: 'apify/instagram-hashtag-scraper',
+    name: `hashtags-${hashtag}`,
+    input: { hashtags: [hashtag], resultsLimit: 20 },
+    options: { memoryMbytes: 1024 },
+}));
+
+// Create all tasks in parallel
+const createdTasks = await Promise.all(socialsTasks);
+
+// Run all tasks in parallel
+await Promise.all(createdTasks.map((task) => client.task(task.id).call()));
+```
+
+## Getting latest data from an Actor, joining datasets
+
+Actor data are stored to [datasets](https://docs.apify.com/platform/storage/dataset). Datasets can be retrieved from Actor runs. Dataset items can be listed with pagination.
+Also, datasets can be merged together.
```

Comment:
Here we must describe the use case properly. Why is it useful to merge datasets together / why should users do it? What is the advantage?